Sources/BraintreePayPalNativeCheckout/BTPayPalNativeCheckoutClient.swift

```diff
@@ -3,6 +3,17 @@ import Foundation
 import BraintreeCore
 #endif
 
-class BTPayPalNativeCheckoutClient: NSObject {
-
+@objcMembers
+public class BTPayPalNativeCheckoutClient: NSObject {
+    private var apiClient: BTAPIClient
+    
+    @objc(initWithAPIClient:)
+    public init(apiClient: BTAPIClient) {
+        self.apiClient = apiClient
+    }
+    
+    public func tokenize(request: BTPayPalNativeCheckoutRequest, completion: (BTPayPalNativeCheckoutNonce?, Error?) -> Void) {
+        // TODO: start native checkout
+    }
 }
+
```

Comment:
nit: it looks like we don't have a newline space in the other files at the end but do here. I don't lean strongly one way or the other as long as we are consistent. For SEPA it looks like we did include a newline space at the end of files. 🤷

Reply:
Yeah, I'm fine with either tbh as long as it's consistent. I think Swiftlint handles this for Unified, but would be nice to stay consistent so if we _do_ switch to a linter post Swift conversion or add a style guide we've already got some standards in place.